charts/aws-ingress/values.yaml

```diff
@@ -0,0 +1,29 @@
+# The following ports must match the ports used in wire-server
+#
+ingress:
+  webapp:
+    https:
+      externalPort: 443
+      sslCert: arn:aws:iam::00000-accountnumber-00000:server-certificate/example.com
```

Comment:
Perhaps you could add a comment here that this is an example and needs to be changed (and link to `[Upload the required certificates](https://aws.amazon.com/premiumsupport/knowledge-center/import-ssl-certificate-to-iam/)`)